test/e2e_node/device_plugin_failures_pod_status_test.go

```diff
@@ -40,7 +39,7 @@ import (
 	"k8s.io/kubernetes/test/e2e_node/testdeviceplugin"
 )
 
-var _ = SIGDescribe("Device Plugin Failures Pod Status", feature.ResourceHealthStatus, func() {
```

Comment:
btw, we also need to remove the feature definition since it was the only usage of it.